src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Color[] COLOR_VALUES = Color.values();
```

Comment:
According to the checklist, don't use static methods in your solution. However, in this context, having a static final array of enum constants is acceptable because it represents a constant set of values that do not change. So this is not an issue.